Pipfile

```diff
@@ -35,5 +35,6 @@ wikibaseintegrator = "==0.11.0"
 ipykernel = "*"
 bs4 = "*"
 pyyaml = "*"
+numpy = "*"
```

Comment:
I think that we shouldn't keep NumPy as a separate dependency since we don't need it except as a requirement for other packages. Should remove this IMO